### ISSUE TITLE
Basic How To Use please

### ISSUE THREAD

**Issue by Exaskryz** [pylerSM/YouTubeBackgroundPlayback#205]
I downloaded this a year or more ago, and it never worked and so I just forgot about it. Now today U wanted to try it again, yet it still doesn't work. Supposedly based on the versions of the apps I have it should be, but it does not.

I must be doing something wrong. What do I have to do for it to work? My process right now is start a video, and press the home or apps universal buttons to put YouTube in the background. No luck on that at all.

**Comment by th3an7** [comment]
- Download the module from xposed repo according to your YT app version (latest module version should work with older youtube)
- Enable module in Xposed App
- Reboot phone
- Start YouTube video in app and make it background (home button or turning screen off should work)
- Profit :P

**Comment by Exaskryz** [comment]
Alright, I had double checked my version numbers to make sure I didn't update to do new of a youtube version, and checked the backgroundplayback as well. Found out that it didn't upgrade the version like I thought I did; got alert fatigued I guess and dismissed the "Installation Error" or whatever message which looked like "App Installed Successfully" that I had just come to expect. So, after uninstalling the old and installing the new app, and rebooting the phone, things worked!

tl;dr user error in installation
